### DISCUSSION TITLE
Can I download onnxruntime.dll 1.14 ? win64

### DISCUSSION THREAD

**Discussion by elephantpanda** [microsoft/onnxruntime#14249] [Other Q&A] [answered]
According to [this article](https://lookanotherblog.com/running-stable-diffusion-image-generation-on-amd-gpu-windows/) it recommends downloading the latest version of DirectML onnxruntime 1.14. But in visual studio the nuget pacakge only goes up to 1.13.
Is there a way to get the latest dll? I want to test to see if it fixes my issues.

**Comment by xadupre** [comment, marked as answer]
onnxruntime 1.14 has not yet been released. I can be manually built.
